roboflow/core/workspace.py

```diff
@@ -99,6 +100,116 @@ def clip_compare(
+                    boundingbox["x"] + boundingbox["width"] / 2,
+                    boundingbox["y"] + boundingbox["height"] / 2,
+                )
+
+                # create a new cropped image using the first stage prediction coordinates (for each box!)
+                croppedImg = pil_image.crop(box)
+                croppedImg.save("./temp.png")
+
+                # capture results of second stage inference from cropped image
+                results.append(stage_two_model.predict("./temp.png")[0])
+        elif stage_one_project.type == "classification":
+            print(
+                "please use an object detection model--can only perform two stage with bounding box results"
+            )
+
+        return results
+
+    def two_stage_ocr(
+        self,
+        image: str = "",
+        first_stage_model_name: str = "",
+        first_stage_model_version: int = 0,
+    ) -> dict:
+        """for each prediction in the first stage object detection, perform OCR as second stage
+        @params:
+            image: (str) = name of the image to be processed
+            first_stage_model: (str) = URL path to the first stage detection model
+            first_stage_model_version: (int) = version number for the first stage model
+
+            returns: (dict) = a json obj containing
+        """
+        results = []
+
+        # create PIL image for cropping
+        pil_image = Image.open(image).convert("RGB")
+
+        # grab first and second stage model from project
+        stage_one_project = self.project(first_stage_model_name)
+        stage_one_model = stage_one_project.version(first_stage_model_version).model
+
+        # perform first inference
+        predictions = stage_one_model.predict(image)
+
+        # interact with each detected object from stage one inference results
+        if stage_one_project.type == "object-detection":
+            for boundingbox in predictions:
+                # rip bounding box coordinates from json1
+                # note: infer returns center points of box as (x,y) and width, height
+                # ----- but pillow crop requires the top left and bottom right points to crop
+                box = (
+                    boundingbox["x"] - boundingbox["width"] / 2,
+                    boundingbox["y"] - boundingbox["height"] / 2,
+                    boundingbox["x"] + boundingbox["width"] / 2,
+                    boundingbox["y"] + boundingbox["height"] / 2,
+                )
+
+                # create a new cropped image using the first stage prediction coordinates (for each box!)
+                croppedImg = pil_image.crop(box)
+
+                # capture OCR results from cropped image
+                results.append(ocr_infer(croppedImg)["results"])
+        else:
+            print(
+                "please use an object detection model--can only perform two stage with bounding box results"
+            )
+
+        return results
```

Comment:
Similar to above, anything we want to do on post processing the results? Should it keep the bounding box info, for example?